CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.0.0-alpha.4 (2019-06-05)
+
+- Manually added support for Chrome 73, 74, and 75. [#9](https://github.com/blackbaud/chromedriver-version-matcher/pull/9)
```

Comment:
Do we need to call out "Manually" here? Doesn't really matter how the support got added, does it?

Reply:
I would like to leave it there as a reminder that I edited the JSON file directly instead of through our script.

Reply:
That's fine. I didn't feel strongly one way or the other.